eth/tools/builder/chain/builders.py

```diff
@@ -446,13 +459,17 @@ def copy(chain: MiningChainAPI) -> MiningChainAPI:
     if isinstance(base_db.wrapped_db, MemoryDB):
         db = AtomicDB(MemoryDB(base_db.wrapped_db.kv_store.copy()))
     else:
-        raise ValidationError(f"Unsupported wrapped database: {type(base_db.wrapped_db)}")
+        raise ValidationError(
+            f"Unsupported wrapped database: {type(base_db.wrapped_db)}"
+        )
 
     chain_copy = type(chain)(db, chain.header)
     return chain_copy
 
 
-def chain_split(*splits: Iterable[Callable[..., Any]]) -> Callable[[ChainAPI], Iterable[ChainAPI]]:   # noqa: E501
+def chain_split(
+    *splits: Iterable[Callable[..., Any]]
+) -> Callable[[ChainAPI], Iterable[ChainAPI]]:  # noqa: E501
```

Comment:
Scan file for unnecessary `# noqa: E501` uses

Reply:
deleted